src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,84 @@
+import { Component } from 'react';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { withStyles } from '@material-ui/core/styles';
+import { arrayOf, object } from 'prop-types';
+import { camelCase } from 'change-case';
+import { memoizeWith } from 'ramda';
+import DataTable from '../DataTable';
+
+@withStyles(() => ({
+  summary: {
+    whiteSpace: 'nowrap',
+  },
+}))
+export default class BugsTable extends Component {
+  static propTypes = {
+    /**
+     * A list of objects to display. Each element in the list is represented
+     * by a row and each element's key-value pair represents a column.
+     */
+    items: arrayOf(object).isRequired,
+  };
+
+  state = {
+    sortBy: null,
+    sortDirection: null,
+  };
+
+  getTableData = memoizeWith(
+    ({ sortBy, sortDirection }) => `${sortBy}-${sortDirection}`,
```

Comment:
We should sort the items and insert it in the memoization key. See https://github.com/taskcluster/taskcluster-web/blob/2998f6561a4de26908d4a2eba4aaca8443c15a53/src/components/RolesTable/index.jsx#L61-L63 For an example.

In the sorted function, instead of returning `roleId` like https://github.com/taskcluster/taskcluster-web/blob/2998f6561a4de26908d4a2eba4aaca8443c15a53/src/components/RolesTable/index.jsx#L25-L28, you can return `${project.project}-${project-summary}`.